requirements.sh

```diff
@@ -64,7 +65,27 @@ if [[ ( $os_name == *"Red Hat"* || $os_name == *"CentOS"* ) &&  $os_version == *
 	autoreconf -f -i
 	make
 	cd $foglamp_location
-	sudo -u $SUDO_USER scl enable rh-python36 bash
```

Comment:
Any specific reason to remove this line?

Reply:
solved